test-data/pod.json

```diff
@@ -64,9 +64,7 @@
         ],
         "dnsPolicy": "ClusterFirst",
         "nodeName": "1.1.1.1",
-        "nodeSelector": {
-            "master": "true"
-        },
+        "nodeSelector": {},
```

Comment:
do we need to say "main: true"?

Reply:
No, this is mocked data and it wasn't being used anyways.